README.md

```diff
@@ -108,7 +108,7 @@ and oauth service to have the same value for them):
 |-------------------------------------------------------|--------------------------------|-----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | --base-url                                            | BASEURL                        |                       | This is the publicly accessible URL on which the SPI OAuth service is reachable. Note that this is not just a hostname, it is a full URL including a scheme, e.g. "https://acme.com/spi"                                           |
 | --config-file                                         | CONFIGFILE                     | /etc/spi/config.yaml  | The location of the configuration file.                                                                                                                                                                                            |
-| --metrics-bind-address                                | METRICSADDR                    | :8080                 | The address the metric endpoint binds to.                                                                                                                                                                                          |
+| --metrics-bind-address                                | METRICSADDR                    | 127.0.0.1:8080        | The address the metric endpoint binds to.                                                                                                                                                                                          |
```

Comment:
While this truly is the default, I'm not sure it is the default that is observable externally because of the rbac-proxy. Maybe we should add a note here that while this is the default from the operator binary point of view, the metrics are still available externally through the authorized endpoint provided by rbac-proxy?